src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -415,76 +427,95 @@ def load_cgf_exposure(key: str, location: str) -> pd.DataFrame:
     data = reshape_to_vivarium_format(data, location)
     return data
 
+
 def load_gbd_2021_exposure(key: str, location: str) -> pd.DataFrame:
-    entity_key = EntityKey(key)
-    entity = utilities.get_gbd_2020_entity(entity_key)
-
-    data = utilities.get_data(entity_key, entity, location, gbd_constants.SOURCES.EXPOSURE, 'rei_id',
-                              metadata.AGE_GROUP.GBD_2021, metadata.GBD_2020_ROUND_ID)
-    data = utilities.process_exposure(data, entity_key, entity, location, metadata.GBD_2020_ROUND_ID,
-                                      metadata.AGE_GROUP.GBD_2021)
-
-    if entity_key == data_keys.STUNTING.EXPOSURE:
-        # Remove neonatal exposure
-        neonatal_age_ends = data.index.get_level_values('age_end').unique()[:2]
-        data.loc[data.index.get_level_values('age_end').isin(neonatal_age_ends)] = 0.0
-        data.loc[data.index.get_level_values('age_end').isin(neonatal_age_ends)
-                 & (data.index.get_level_values('parameter') == data_keys.STUNTING.CAT4)] = 1.0
+    key = EntityKey(key)
+    entity = utilities.get_gbd_2021_entity(key)
+    location_id = (
+        utility_data.get_location_id(location) if isinstance(location, str) else location
+    )
+    data = utilities.get_data(
+        key,
+        entity,
+        location,
+        gbd_constants.SOURCES.EXPOSURE,
+        "rei_id",
+        metadata.AGE_GROUP.GBD_2021,
+        metadata.GBD_2021_ROUND_ID,
+    )
+    data = data.drop(["modelable_entity_id", "model_version_id"], "columns")
```

Comment:
What changed such that you couldn't just call `utilities.process_exposure`?

Reply:
The normalization that was happening in `process_exposure` generated data that didn't look right; I'm not 100% sure what was going wrong, but in its place I just used the relevant portion of `get_exposure_without_model_version_id`